cg/meta/workflow/balsamic.py

```diff
@@ -92,20 +88,13 @@ def get_case_path(self, case_id: str) -> Path:
         """Returns a path where the Balsamic case for the case_id should be located"""
         return Path(self.root_dir, case_id)
 
-    def get_cases_to_analyze(self) -> list[Case]:
-        cases_query: list[Case] = self.status_db.cases_to_analyze(
-            workflow=self.workflow, threshold=self.use_read_count_threshold
-        )
-        cases_to_analyze = []
-        for case_obj in cases_query:
-            if case_obj.action == "analyze" or not case_obj.latest_analyzed:
-                cases_to_analyze.append(case_obj)
-            elif (
-                self.trailblazer_api.get_latest_analysis_status(case_id=case_obj.internal_id)
-                == "failed"
-            ):
-                cases_to_analyze.append(case_obj)
-        return cases_to_analyze
+    def get_cases_ready_for_analysis(self) -> list[Case]:
+        """Returns a list of cases that are ready for analysis."""
+        cases_to_analyze: list[Case] = self.get_cases_to_analyse()
+        cases_ready_for_analysis: list[Case] = [
+            case for case in cases_to_analyze if self.is_case_ready_for_analysis(case)
+        ]
+        return cases_ready_for_analysis
```

Comment:
I guess it is because the function does more than that. it gets cases ready for analysis, which is dependent on sequencing qc, but also other factors inside the `self.is_case_ready_for_analysis(case)`